docs/template_plugin/README.md

```diff
@@ -1,19 +1,19 @@
-# template-plugin
+# OpenVINO Template Plugin
 
-Template Plugin for Inference Engine which demonstrates basics of how Inference Engine plugin can be built and implemented on top of Inference Engine Developer Package and Plugin API.
-As a backend for actual computations ngraph reference implementations is used, so the Template plugin is fully functional.
+Template Plugin for OpenVINO™ Runtime which demonstrates basics of how OpenVINO™ Runtime plugin can be built and implemented on top of OpenVINO Developer Package and Plugin API.
+As a backend for actual computations OpenVINO reference implementations is used, so the Template plugin is fully functional.
 
 ## How to build
 
 ```bash
-$ cd $DLDT_HOME
-$ mkdir $DLDT_HOME/build
-$ cd $DLDT_HOME/build
+$ cd $OPENVINO_HOME
```

Comment:
Usage of an undefined variable in the readme file explaining how to build this plugin is confusing. It should either be defined in this code snippet or replaced with simple `openvino` directory. Perhaps the first command in this snippet should be the `git clone` of the OV repo?

Reply:
Changed variables $.+ to <.+> in order to avoid collisions with environment variables.